libraries/botbuilder-ai-orchestrator/src/orchestratorRecognizer.ts

```diff
@@ -309,24 +317,24 @@ export class OrchestratorRecognizer extends AdaptiveRecognizer implements Orches
         // eslint-disable-next-line  @typescript-eslint/no-unused-vars
         const { text, alteredText, intents, entities, ...customRecognizerProps } = recognizerResult;
         const properties: Record<string, string> = {
-            TopIntent: intent.length > 0 ? topTwo[0].name : undefined,
-            TopIntentScore: intent.length > 0 ? topTwo[0].score.toString() : undefined,
-            NextIntent: intent.length > 1 ? topTwo[1].name : undefined,
-            NextIntentScore: intent.length > 1 ? topTwo[1].score.toString() : undefined,
-            Intents: intent.length > 0 ? JSON.stringify(recognizerResult.intents) : undefined,
-            Entities: recognizerResult.entities ? JSON.stringify(recognizerResult.entities) : undefined,
+            TopIntent: intent.length > 0 ? topTwo[0].name : '',
+            TopIntentScore: intent.length > 0 ? topTwo[0].score.toString() : '',
+            NextIntent: intent.length > 1 ? topTwo[1].name : '',
+            NextIntentScore: intent.length > 1 ? topTwo[1].score.toString() : '',
+            Intents: intent.length > 0 ? JSON.stringify(recognizerResult.intents) : '',
+            Entities: recognizerResult.entities ? JSON.stringify(recognizerResult.entities) : '',
             AdditionalProperties: JSON.stringify(customRecognizerProps),
         };
 
         let logPersonalInformation =
             this.logPersonalInformation instanceof BoolExpression
-                ? this.logPersonalInformation.getValue(dialogContext.state)
+                ? this.logPersonalInformation.getValue((dialogContext as DialogContext).state)
```

Comment:
` as DialogContext` should be unnecessary.

Reply:
Some other lint and environment asked for it, though not here. Just try to appease to both lint.